### ISSUE TITLE
Python 2.6 has been broken

### ISSUE THREAD

**Issue by KonstantinShemyak** [obriencj/python-javatools#50]
The following features are not available in Python 2.6: 

- Commit b354df8cba8a58136f15110e7d6d8a3e5df4069c (from year 2012) used `zipfile.ZipFile` as context manager.
- Commit 342af4973bfd62 introduced line with multiple `with` contexts into `jardiff.py`.
- Commit 675987d59915 imports `OrderedDict`.
- Multiple commits added `assertIsNone()`/`assertIsNotNone()` to tests.
- `assertRaises()` is used as context manager.
- `argparse` is not avilable in 2.6.

**Comment by KonstantinShemyak** [comment]
@obriencj So 2.6 support has been broken since 2012, and then I have beaten it down significantly more. Do we want to maintain it? All issues I identified can be fixed, but the price is language clarity.

**Comment by JavierMartin68** [comment]
Hello Konstantin.
I came to this project searching for a tool to easily maintain MANIFEST,MF from several jars using python 2.7.
I have opened jar files using zipfile and obtaining MANIFEST.MF and changing some parts, but it is not as easy to change a lot of sections like Import-Package if there are several lines trunked by 70 characters.
I will try using this python-javatools as it seems to be very well produced! Thanks.
